### ISSUE TITLE
[Accessibility]: StackedBarChart graph SVG element has no accessible name

### ISSUE THREAD

**Issue by rileydanejohnston** [carbon-design-system/carbon-charts#1975]
### Application/Team

billing

### Environment

Browser

### What happened? What did you expect to happen instead?

Original issue: https://github.ibm.com/ibmcloud/billing/issues/3387

Page URL: https://test.cloud.ibm.com/billing

Recommended fix: The SVG element has no accessible name. No alternative text is provided for the graph representation

User impact: Users with visual impairments cannot understand the graph's content due to the lack of alternative text for the SVG element.

WCAG Mapping: 1.1.1 Non-Text Content

Level of Compliance: A

Severity: Medium

Screen print:

![Image](https://github.com/user-attachments/assets/63b45a5f-7366-42ce-a40e-6ac6616dab68)

### What WCAG 2.1 checkpoint does the issue violate?

Non-Text Content

### Version

"@carbon/charts-react": "^1.22.18",

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2

**Comment by theiliad** [comment]
A fix for this was actually rolled out minutes ago, should be live soon

**Comment by rileydanejohnston** [comment]
Hi @theiliad, I bumped `@carbon/charts` and `@carbon/charts-react` to v1.23.7 and I'm still seeing the issue in the IBM Equal Access Accessibility Checker. What version can I find the fix on? Here's what I see when I run the microservice locally:

<img width="800" alt="Image" src="https://github.com/user-attachments/assets/3ff06086-cf06-403c-a9ba-e08e7616d3e8" />
